tests/apps/courses/test_templates_program_detail.py

```diff
@@ -183,3 +184,105 @@ def test_templates_program_detail_cms_published_content_opengraph(self):
+
+        self.assertContains(
+            response,
+            '<meta name="description" content="A custom description of the program" />',
+        )
+
+    def test_templates_program_detail_meta_description_program_excerpt(self):
+        """
+        The program meta description should show the program_excerpt if no meta_description is
+        specified
+        """
+        program = ProgramFactory()
+        page = program.extended_object
+
+        # Add a excerpt to a program
+        placeholder = program.extended_object.placeholders.get(slot="program_excerpt")
+        add_plugin(
+            language="en",
+            placeholder=placeholder,
+            plugin_type="PlainTextPlugin",
+            body="A program excerpt description",
+        )
+        page.publish("en")
+
+        url = program.extended_object.get_absolute_url()
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+
+        self.assertContains(
+            response,
+            '<meta name="description" content="A program excerpt description" />',
+        )
+
+    def test_templates_program_detail_meta_description_program_excerpt_exceeds_max_length(
+        self,
+    ):
+        """
+        The program meta description should show the program_excerpt if no meta_description is
+        specified
+        """
+        program = ProgramFactory()
+        page = program.extended_object
+        placeholder_value = (
+            "Long description that describes the page with a summary. "
+            "Long description that describes the page with a summary. "
+            "Long description that describes the page with a summary. "
+        )
+
+        # Add a excerpt to a program
+        placeholder = program.extended_object.placeholders.get(slot="program_excerpt")
+        add_plugin(
+            language="en",
+            placeholder=placeholder,
+            plugin_type="PlainTextPlugin",
+            body=placeholder_value,
+        )
+        page.publish("en")
+
+        url = program.extended_object.get_absolute_url()
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+
+        cut = placeholder_value[0:160]
+        self.assertContains(
+            response,
+            f'<meta name="description" content="{cut}" />',
+        )
+
+    def test_templates_program_detail_meta_description_empty(self):
+        """
+        The program meta description should not be present if neither the meta_description field
+        on the page, nor the `program_excerpt` placeholder are filled
+        """
+        program = ProgramFactory()
+        page = program.extended_object
+        page.publish("en")
+
+        url = program.extended_object.get_absolute_url()
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+
+        self.assertNotContains(
+            response,
+            '<meta name="description"',
+        )
```

Comment:
You forgot to add the same tests for the organization

Reply:
Now, I've included them.